### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mattlub**
related #11

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,12 @@
 # nazareth-marketing
-Nazareth Marketing Committee discussions and decisions
+Repository for the Nazareth marketing committee, containing discussions and decisions with regards to the marketing strategy of Founders and Coders in Nazareth.
+
+### Key links:
+- The [charter](https://github.com/foundersandcoders/nazareth-marketing/blob/master/committee-charter.md)
+
+### Next scheduled meeting:
+3/4/17
```
Comment (rebeccaradding) [inline README.md]: Our next meeting is in May.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,12 @@
...
+
+### Social media links:
+  - Founders and Coders Nazareth [facebook page](https://www.facebook.com/founderscodersnazareth/)
```
Comment (rebeccaradding) [inline README.md]: Facebook should be capitalized. Also, add Twitter (@founderscoders).

**Comment by mattlub** [comment]
thanks @rebeccaradding, updated now

**Comment by rebeccaradding** [comment]
@mattlub happy with this, @m4v15 and @Suuuuha what do you think?

**Pull request merged**